vim/yanking-text-to-register.md

```diff
@@ -0,0 +1,21 @@
+# Yanking text to register
+
+Yanking and deleting text will save the text to the same unnamed register. Which means every time you delete a word, your 
+yanked word will be overwritten. Which caused me endless amounts of frustration when I accidentally deleted a word and had to
```

Comment:
Yup 'this' makes more sense. Fixed in commit 1a3888ced8c199079df6ee67da5de4fb9dc50cef.